js/src/forum/components/CommentPost.js

```diff
@@ -72,7 +73,7 @@ export default class CommentPost extends Post {
   }
 
   isEditing() {
-    return app.composer.component instanceof EditPostComposer && app.composer.component.props.post === this.props.post;
+    return app.composer.bodyClass && subclassOf(app.composer.bodyClass, EditPostComposer) && app.composer.bodyProps.post === this.props.post;
```

Comment:
We could encapsulate this as something like `app.composer.bodyIs(componentClass, props)`.